accesscontrol/jwt.go

```diff
@@ -252,14 +240,7 @@ func newParser(algo Algorithm, claims map[string]interface{}) (*jwt.Parser, erro
 func parsePublicPEMKey(key []byte) (pub *rsa.PublicKey, err error) {
 	pemBlock, _ := pem.Decode(key)
 	if pemBlock == nil {
-		decKey, err := base64.StdEncoding.DecodeString(string(key))
```

Comment:
Why the base64 fallback gets removed? A user may reference a base64 encoded secret (env). This enforces explicit usage of our `base64_decode` func. If it is a decoding error we can still throw the mustPEMencoded one.

Reply:
There seems to be no known use-case. We will remove this behaviour now.